src/pages/PagePrivacyPolicy.vue

```diff
@@ -0,0 +1,183 @@
+<template lang="pug">
+  .PageTerms.terms
+    h2
+      strong Residual Token Inc. Privacy Policy
+    p
+      | Residual Token Inc. (&quot;Residual Token&quot;, &ldquo;We&rdquo; or &ldquo;Our&rdquo;) is committed to protecting the privacy of users of its software applications and services. This Privacy Policy states the Residual Token practices applying to individual users of the Residual Token managed services (the &quot;Services&quot;). Residual Token owns, operates, and maintains the Services. Residual Token is a Delaware corporation with its principal office in New York, NY.
```

Comment:
@mkwatson can you pls add line breaks?

Reply:
@ColtMoney Done. We should find a formatter!!! Do you want me to squash these two commits into one?